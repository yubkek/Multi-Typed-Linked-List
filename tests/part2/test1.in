NEW 3
1
c
32e2
NEW 4
a
b
character
324
INSERT 1 -45 what
INSERT 0 0 LOL
