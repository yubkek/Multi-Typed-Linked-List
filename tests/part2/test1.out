List 0: 1 -> c -> 3200.00
List 1: a -> b -> character -> 324
List 1: a -> b -> character -> 324 -> what
List 0: LOL -> 1 -> c -> 3200.00